extensions/amp-story/amp-story-auto-ads.md

```diff
@@ -16,39 +16,75 @@ limitations under the License.
 
 # <a name="`amp-story-auto-ads`"></a> `amp-story-auto-ads`
 
-
 ## THIS IS A WORK-IN-PROGRESS
 
 <table>
   <tr>
     <td width="40%"><strong>Description</strong></td>
-    <td>FILL THIS IN</td>
+    <td>Dynamically inserts ads into a Story.</td>
   </tr>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td>FILL THIS IN</td>
+    <td>Experimental</td>
   </tr>
   <tr>
     <td width="40%"><strong>Required Script</strong></td>
-    <td><code>&lt;script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-story-auto-ads-0.1.js">&lt;/script></code></td>
+    <td><code>&lt;script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-0.1.js">&lt;/script></code></td>
   </tr>
   <tr>
     <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
-    <td>FILL THIS IN</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>Examples</strong></td>
-    <td>FILL THIS IN</td>
+    <td>N/A</td>
   </tr>
 </table>
 
 ## Behavior
+`amp-story-auto-ads` extension dynamically inserts ads (implemented as `amp-ad`)
+into the story at the time when the content is consumed.
+
+Each `amp-ad` is inserted as a full screen story page. To prevent showing 
+blank/unloaded ads, the ad is pre-rendered completely in the background before 
+being visible to users. Based on user interactions, the extension decides when 
+and where to insert ads.
+
+Ad in story can be skipped the same way as normal story pages by tapping on the
+right part of the screen.
+
+## Configuration
+In the `<amp-story-auto-ads>` element, you specify a JSON configuration object 
+that contains the details for how ads should be fetched and displayed, which
+looks like the following:
+
+```html
+<amp-story>
+  <amp-story-auto-ads>
+     <script type="application/json">
+       {
+          "ad-attributes": {
+            "type": "custom",
+            "data-src": "https://adserver.com/getad?slot=abcd1234"
```

Comment:
Can you talk a bit about what data-src is? what should publishers do with it?

Reply:
added a link to custom ad spec for details.